src/App.tsx

```diff
@@ -6,19 +7,22 @@ import './assets/styles/vars.css';
 import './assets/styles/typography.css';
 
 function App() {
+  const genres = ['All', 'Documentary', 'Comedy', 'Horror', 'Crime'];
```

Comment:
you can create ENUM for values and file with consts for re-using them
`enum Genres {All = 'All', Documentary = 'Documentary', ...}
const genres = [Genres.All, Genres.Documentary, ...]`